go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/reddit/baseplate.go
 
-go 1.20
+go 1.21
```

Comment:
I think we should wait until 1.22 is fully released.  I don't expect 1.22 until next month, right?

Reply:
I disagree that baseplate.go needs to follow go release cycle strictly. if some service is still using go 1.20, they are _very unlikely_ to upgrade baseplate.go version anyways, and if upgrading baseplate.go version breaks them it's time for them to upgrade to go 1.21 as well.

we certainly should not drop support to a go version as soon as the next version is released, but dropping support at the end of the lifecycle to a go version is appropriate. 1.20 is also unlike 1.19 that the next version has some performance regressions.

Reply:
Well, the baseplate team _does_ want to follow the upstream Go release cycle :).  Maybe not "strictly," but definitely not _more_ strictly than the upstream.